dietpi/dietpi-logclear

```diff
@@ -231,6 +231,19 @@
 		#delete[] array
 		unset ARRAY_LOG_FILEPATH
 
+		#Update dietpi-ramlog file structure, if enabled:
+ 		LOGGING_MODE=0
+ 		if [[ -r '/DietPi/dietpi/.installed' ]]; then
```

Comment:
```-r```? 😕

Reply:
@Fourdee 
File is readable 😄: https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html